src/DssSpell.sol

```diff
@@ -104,289 +110,295 @@ contract SpellAction {
+        // !!!!!!!! Only if PIP_AAVE = Osm, its src is a Median and hasn't been already whitelisted due a previous deployed ilk
+        MedianAbstract(OsmAbstract(PIP_AAVE).src()).kiss(PIP_AAVE);
         // Whitelist Spotter to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_UNI = Osm or PIP_UNI = Median and hasn't been already whitelisted due a previous deployed ilk
-        OsmAbstract(PIP_UNI).kiss(MCD_SPOT);
+        // !!!!!!!! Only if PIP_AAVE = Osm or PIP_AAVE = Median and hasn't been already whitelisted due a previous deployed ilk
+        OsmAbstract(PIP_AAVE).kiss(MCD_SPOT);
         // Whitelist End to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_UNI = Osm or PIP_UNI = Median and hasn't been already whitelisted due a previous deployed ilk
-        OsmAbstract(PIP_UNI).kiss(MCD_END);
-        // Set UNI Osm in the OsmMom for new ilk
-        // !!!!!!!! Only if PIP_UNI = Osm
-        OsmMomAbstract(OSM_MOM).setOsm(ILK_UNI_A, PIP_UNI);
-
-        // Set the UNI-A debt ceiling
-        VatAbstract(MCD_VAT).file(ILK_UNI_A, "line", 15 * MILLION * RAD);
-        // Set the UNI-A dust
-        VatAbstract(MCD_VAT).file(ILK_UNI_A, "dust", 500 * RAD);
+        // !!!!!!!! Only if PIP_AAVE = Osm or PIP_AAVE = Median and hasn't been already whitelisted due a previous deployed ilk
+        OsmAbstract(PIP_AAVE).kiss(MCD_END);
+        // Set AAVE Osm in the OsmMom for new ilk
+        // !!!!!!!! Only if PIP_AAVE = Osm
+        OsmMomAbstract(OSM_MOM).setOsm(ILK_AAVE_A, PIP_AAVE);
+
+        // Set the AAVE-A debt ceiling
+        VatAbstract(MCD_VAT).file(ILK_AAVE_A, "line", 10 * MILLION * RAD);
+        // Set the AAVE-A dust
+        VatAbstract(MCD_VAT).file(ILK_AAVE_A, "dust", 500 * RAD);
         // Set the Lot size
-        CatAbstract(MCD_CAT).file(ILK_UNI_A, "dunk", 50 * THOUSAND * RAD);
-        // Set the UNI-A liquidation penalty (e.g. 13% => X = 113)
-        CatAbstract(MCD_CAT).file(ILK_UNI_A, "chop", 113 * WAD / 100);
-        // Set the UNI-A stability fee (e.g. 1% = 1000000000315522921573372069)
-        JugAbstract(MCD_JUG).file(ILK_UNI_A, "duty", THREE_PERCENT_RATE);
-        // Set the UNI-A percentage between bids (e.g. 3% => X = 103)
-        FlipAbstract(MCD_FLIP_UNI_A).file("beg", 103 * WAD / 100);
-        // Set the UNI-A time max time between bids
-        FlipAbstract(MCD_FLIP_UNI_A).file("ttl", 6 hours);
-        // Set the UNI-A max auction duration to
-        FlipAbstract(MCD_FLIP_UNI_A).file("tau", 6 hours);
-        // Set the UNI-A min collateralization ratio (e.g. 150% => X = 150)
-        SpotAbstract(MCD_SPOT).file(ILK_UNI_A, "mat", 175 * RAY / 100);
-
-        // Update UNI-A spot value in Vat
-        SpotAbstract(MCD_SPOT).poke(ILK_UNI_A);
+        CatAbstract(MCD_CAT).file(ILK_AAVE_A, "dunk", 50000 * RAD);
```

Comment:
For consistency, I'd express it as `50 * THOUSAND * RAD`.

Reply:
Fixed